README.md

```diff
@@ -1,179 +1,46 @@
-# 🚀 api
+<p align="center">
+  <img width="350" src="./docs/images/logo.svg" />
```

Comment:
I'll update these relative URLs to the raw Github links after this is merged into `main`.